PowerUp/app/src/main/java/powerup/systers/com/ui/store_screen/StoreActivity.java

```diff
@@ -35,6 +35,9 @@
 import powerup.systers.com.utils.PowerUpUtils;
 
 import static powerup.systers.com.utils.PowerUpUtils.MAX_ELEMENTS_PER_SCREEN;
+import static powerup.systers.com.utils.PowerUpUtils.TYPE_ACCESSORIES;
+import static powerup.systers.com.utils.PowerUpUtils.TYPE_CLOTHES;
+import static powerup.systers.com.utils.PowerUpUtils.TYPE_HAIR;
 
```

Comment:
I have used static imports so that "PowerUpUtils" is not needed to be used again and again.